src/components/Heading/Heading.js

```diff
@@ -0,0 +1,54 @@
+import PropTypes from "prop-types";
+import React from "react";
+
+export function Header(props) {
+  const { headingLevel, title, fromLink, fromText } = props;
+  const Title = headingLevel;
+
+  return (
+    <>
+      <Title>{title}</Title>
+      {fromLink && fromLink && (
+        <p className="gc-byline">
+          <strong>From: </strong>
+          <a href={fromLink}>{fromText}</a>
+        </p>
+      )}
```

Comment:
I believe the fromLink only applies to the heading 1. So heading 2,3,4,5,6 don't have a fromLink